meta-mender-demo/recipes-core/images/core-image-full-cmdline.bbappend

```diff
@@ -1 +1 @@
-IMAGE_INSTALL_append = " hello-mender"
+IMAGE_INSTALL_append = " hello-mender boot-script"
```

Comment:
> Is there a better way to "include" it than in the file core-image-full-cmdline.bbappend?
> Would be nice if it was not specific to bitbake build target.

Yes, I believe so. In fact, I think we should move both, so open the `meta-mender-demo/recipes-core/packagegroups/packagegroup-core-boot.bbappend`, and add it like this:

```
RDEPENDS_${PN}_append = " hello-mender boot-script"
```

and then just delete the `core-image-full-cmdline.bbappend` file.